dev-tools/import_dashboards.sh

```diff
@@ -171,10 +171,10 @@ fi
 if [ -d "${DIR}/index-pattern" ]; then
     for file in ${DIR}/index-pattern/*.json
     do
-        NAME=`awk '$1 == "\"title\":" {gsub(/[",]/, "", $2); print $2}' ${file}`
```

Comment:
I wouldn't remove this as it gets the name of the index pattern from the `title` field available in the json file, that can be different than the name of the file.  Think of `packetbeat-*` that is available under packetbeat.json